services/svc/batch/examples/create_task.rs

```diff
@@ -31,12 +31,12 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     println!("creating job");
     let job_params = JobAddParameter::new(job_id.clone(), pool_info);
-    client.job_client().add(job_params).into_future().await?;
+    client.job_client().add(job_params).send().await?;
```

Comment:
If you don't need the response body, just call `send`. An `into_body` is only available if there is a body.